custom/enikshay/__init__.py

```diff
@@ -1 +1,10 @@
 from custom.enikshay.integrations.ninetyninedots.repeater_generators import RegisterPatientPayloadGenerator
+from custom.enikshay.reports.case_finding import CaseFindingReport
+from custom.enikshay.reports.treatment_outcome import TreatmentOutcomeReport
+
+CUSTOM_REPORTS = (
```

Comment:
I think we are trying to move these into `reports.py` moving forwards to avoid too many imports in the `__init__` modules